TPMSAD-1610:In SAD layout add dropdown for languages, also Nepalis an…

poghosyandav opened the pull request:
…d Amharic langues to the existing list

Added dropDownDisplay to have option for showing languages as dropdown list.

poghosyandav committed:
TPMSAD-1610:In SAD layout add dropdown for languages, also Nepalis and Amharic langues to the existing list
Add dropDownDisplay to have option for showing languages as dropdown list.

bdbogjoe commented on grails-app/views/langSelector/_selector.gsp:
```diff
@@ -1,21 +1,13 @@
-<g:set var="locales" value="${(List<Locale>)locales}" />
-<g:set var="selected" value="${(Locale)selected}" />
+<%@ page import="grails.util.Holders; grails.plugins.GrailsPluginManager" %>
 <g:set var="hasAsset"
```
please remove this hasAsset, not needed any more in this file

poghosyandav replied:
you mean remove it at all and don't use it ?, cause I just kept it in this general template to use in child display(bar/dropdown) templates.

bdbogjoe replied:
still used in other templates, but not in this one

poghosyandav replied:
ok, have done changes

poghosyandav committed:
TPMSAD-1610:In SAD layout add dropdown for languages, also Nepalis and Amharic langues to the existing list
removed hasAssest var from selector template and add it in barDisplay and dropDownDisplay templates
